package.json

```diff
@@ -121,6 +121,7 @@
     "react-helmet": "^5.2.0",
     "react-router-config": "^4.4.0-beta.6",
     "react-router-dom": "^4.3.1",
+    "recompose": "^0.30.0",
```

Comment:
As its advised not to use recompose, and I doubt maintainance will be high on anyones list, we should pull out this teeny function into simorgh helpers, with credit ofcourse :P https://github.com/acdlite/recompose/blob/master/src/packages/recompose/compose.js

Reply:
done.